app/main/hotkeys.js

```diff
@@ -44,16 +42,40 @@ const HOTKEYS = new Map([
   ['CmdOrCtrl+0', bw => () => bw.webContents.send('zoom-reset')],
 ]);
 
+const CUSTOM_HANDLED_HOTKEYS = new Map([
+  ['CmdOrCtrl+Left', bw => () => bw.webContents.send('go-back')],
+  ['CmdOrCtrl+Right', bw => () => bw.webContents.send('go-forward')],
+]);
+
+const handlerSet = new Set();
+
 export function bindBrowserWindowHotkeys(browserWindow) {
+  const handler = handleGuestActiveElementChange.bind(null, browserWindow);
+  handlerSet.add(handler);
+  ipcMain.on('guest-active-element', handler);
+
   HOTKEYS.forEach((handlerFactory, accelerator) => {
     shortcut.register(browserWindow, accelerator, handlerFactory(browserWindow));
   });
 }
 
 export function unbindBrowserWindowHotkeys(browserWindow) {
+  handlerSet.forEach(handler => ipcMain.removeListener('guest-active-element', handler));
   shortcut.unregisterAll(browserWindow);
 }
 
+function handleGuestActiveElementChange(browserWindow, event, { details }) {
+  if (!details || details.nodeName !== 'INPUT') {
+    CUSTOM_HANDLED_HOTKEYS.forEach((handlerFactory, accelerator) => {
+      shortcut.register(browserWindow, accelerator, handlerFactory(browserWindow));
```

Comment:
You'd think that were the case, but I've tried that and no. If it were that easy, I could've handled all of this in the frontend without needing to send the currently active element to the main process at all (prevent the actions from happening there).

In short, registered shortcuts swallow the default from happening even when they don't do anything. So if `CmdOrCtrl+Left` is bound, even though the handler does nothing, there won't be anything going on in the input element at all.